### PR TITLE
DOC-3985-CD RN 1.26.x

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

**Review by krishi0408 — DISMISSED** [review]

**Review by SushrutHarness — DISMISSED** [review]
LGTM

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Review by krishi0408 — APPROVED** [review]

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -49,6 +49,43 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+  
+  This issue occurred due to a `SocketTimeoutException: Read timed out` exception when calculating reports per day or month. The queries timed out, and the fallback to generate the data also timed out. 
+
+  This issue is fixed by redefining the functions to calculate license usage for a given date instead of a date range. The heavy SELECT queries are now moved to the secondary time series database (TSDB). This fixed the timeout issues in the license trends query. 
+- Helm manifest runtime inputs for chart versions appear as runtime input in the pipeline execution page, even if the chart version was already provided as a fixed value. (CDS-89158)
+  
+  This issue is fixed. 
+- The Continue button in the Services section was enabled even when the selected service was not available. (CDS-85658)
+  
+  This issue is fixed by adding form validation to the service field when:
```
Comment (shylajasun) [inline release-notes/continuous-delivery.md:99]: @roshnisarangadharan this issue is not clear on the remote service

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -49,6 +49,43 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+  This issue occurred because the string-to-time conversion function in the ANTLR based DSL language did not accept epoch time as a valid input. This issue is now fixed. 
+- The CD Activity & Usage page's Trend tab was not working properly for some accounts. (CDS-89394)
+  
+  This issue occurred due to a `SocketTimeoutException: Read timed out` exception when calculating reports per day or month. The queries timed out, and the fallback to generate the data also timed out. 
+
+  This issue is fixed by redefining the functions to calculate license usage for a given date instead of a date range. The heavy SELECT queries are now moved to the secondary time series database (TSDB). This fixed the timeout issues in the license trends query. 
+- Helm manifest runtime inputs for chart versions appear as runtime input in the pipeline execution page, even if the chart version was already provided as a fixed value. (CDS-89158)
+  
+  This issue is fixed. 
+- The Continue button in the Services section was enabled even when the selected service was not available. (CDS-85658)
```
Comment (shylajasun) [inline release-notes/continuous-delivery.md:97]: @roshnisarangadharan Can you mention this is for remote services

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -49,6 +49,43 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+  These connectors did not check the connectivity to the Secrets Manager. Hence, the secrets were not getting resolved on the delegate. This issue is fixed. Now, the connection test also checks if the Secret Manager is accessible.
+- Harness did not display dynamically provisioned infrastructure inputs when the input field was set as runtime. (CDS-90757)
+  
+  This issue is fixed. The provisioner runtime input field now appears in the infrastructure input card.
+- Service variable description did not appear for template input sets and deployment input sets during pipeline execution. (CDS-89650, ZD-57312, ZD-58330)
+  
+  This issue is fixed.
+- Log verification step failed due to a parsing error. (CDS-89622)
+  
+  This issue occurred because the string-to-time conversion function in the ANTLR based DSL language did not accept epoch time as a valid input. This issue is now fixed.
```
Comment (shylajasun) [inline release-notes/continuous-delivery.md:88]: @roshnisarangadharan Can this line be rephrased

File: release-notes/continuous-delivery.md [hunk 4]
```diff
@@ -49,6 +49,43 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+  
+  The date API call got cancelled when the `useEffect` component was being loaded causing this issue. This issue is fixed by making the date API call only after the `useEffect` component was mounted. 
+
+  The error handling is also improved by displaying a proper error message as part of the failed API response. On instances where a proper message is not present, `Something went wrong` message appears by default. 
+- Artifactory and Git connectors did not honor Secrets Manager selector. (CDS-91300, ZD-57541)
+
+  These connectors did not check the connectivity to the Secrets Manager. Hence, the secrets were not getting resolved on the delegate. This issue is fixed. Now, the connection test also checks if the Secret Manager is accessible.
+- Harness did not display dynamically provisioned infrastructure inputs when the input field was set as runtime. (CDS-90757)
+  
+  This issue is fixed. The provisioner runtime input field now appears in the infrastructure input card.
```
Comment (shylajasun) [inline release-notes/continuous-delivery.md:82]: There are some formatting issues . it is not clear which update release is for which issue

File: release-notes/continuous-delivery.md (hunk 3 above)
Comment (roshnisarangadharan) [reply release-notes/continuous-delivery.md:88]: @shylajasun could you please suggest what needs to be changes?

File: release-notes/continuous-delivery.md [hunk 5]
```diff
@@ -49,6 +49,57 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+  This issue occurred because the string-to-time conversion function in the ANTLR based DSL language did not accept epoch time as a valid input. This issue is now fixed. 
+- The CD Activity & Usage page's Trend tab was not working properly for some accounts. (CDS-89394)
+  
+  This issue occurred due to a `SocketTimeoutException: Read timed out` exception when calculating reports per day or month. The queries timed out, and the fallback to generate the data also timed out. 
+
+  This issue is fixed by redefining the functions to calculate license usage for a given date instead of a date range. The heavy SELECT queries are now moved to the secondary time series database (TSDB). This fixed the timeout issues in the license trends query. 
+- Helm manifest runtime inputs for chart versions appear as runtime input in the pipeline execution page, even if the chart version was already provided as a fixed value. (CDS-89158)
+  
+  This issue is fixed. 
+- The Continue button in the Services section was enabled even when the selected service was not available. (CDS-85658)
```
Comment (roshnisarangadharan) [inline release-notes/continuous-delivery.md:97]:
```suggestion
- The Continue button in the Services section for remote services was enabled even when the selected service was not available. (CDS-85658)
```

File: release-notes/continuous-delivery.md (hunk 1 above)
Comment (roshnisarangadharan) [reply release-notes/continuous-delivery.md:99]: @shylajasun remote service is mentioned below.

**Review by SudarshanPurohit — APPROVED** [review]

File: release-notes/continuous-delivery.md [hunk 6]
```diff
@@ -49,6 +49,57 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+- Artifactory and Git connectors did not honor Secrets Manager selector. (CDS-91300, ZD-57541)
+
+  These connectors did not check the connectivity to the Secrets Manager. Hence, the secrets were not getting resolved on the delegate. This issue is fixed. Now, the connection test also checks if the Secrets Manager is accessible. 
+  
+  This fix is behind the feature flag, `CDS_SERVICE_AND_INFRA_STEP_DELEGATE_SELECTOR_PRECEDENCE`. To enable this fix, contact [Harness Support](mailto:support@harness.io). 
+- Users were unable to create custom queries that are nor part of the APM metrics as a heath source for monitored services. (CDS-91181, ZD-57562)
+  
+  This issue is fixed by making the service instance field configurable for users.
+
+  This item requires Harness Delegate version 82402. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```
Comment (roshnisarangadharan) [inline release-notes/continuous-delivery.md:78]:
```suggestion
  This item requires Harness Delegate version 24.02.82402. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

**Pull request merged**